src/app/store/collectibles/reducer.ts

```diff
@@ -11,7 +11,7 @@ export const collectiblesReducer = createReducer<CollectiblesState>(collectibles
   });
   builder.addCase(loadCollectiblesDetailsActions.success, (state, { payload }) => ({
     ...state,
-    details: createEntity(payload)
+    details: createEntity({ ...state.details.data, ...payload })
```

Comment:
I'm afraid that the store may contain plenty of irrelevant collectibles after this change